napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -205,7 +205,7 @@ def add_rectangle(layer: Shapes, event: MouseEvent) -> None:
     event: MouseEvent
         A proxy read only wrapper around a vispy mouse event.
     """
-    size = layer._vertex_size * layer.scale_factor / 4
+    size = layer._normalized_vertex_radius / 2
```

Comment:
here is division by 2 and in definition is division by 2. is it mistake?

Reply:
it used to be `/ 4`, so it adds up. Why exactly it was `/ 4` I'm not sure, I haven't digged into that, but it's unchanged and it works :P